docs/observability/ovn-observability.md

```diff
@@ -0,0 +1,107 @@
+# Observability
+
+## Introduction
+
+Observability feature uses OVN sampling functionality to generate samples with requested metadata when
+specific OVS flows are matched. To see the generated samples, a binary called `ovnkube-observ` is used.
+This binary allows printing the samples to stdout or writing them to a file.
+
+Currently, supports observability for:
+- Network Policy
+- (Baseline) Admin Network Policy
+- Egress firewall
+- UDN isolation
+- Multicast ACLs
+
+More features are planned to be added in the future. 
+
+## Motivation
+
+Networking observability is an important feature to verify the expected networking behavior in a cluster and
+to debug existing problems.
+Ovn-kubernetes makes use of many abstraction layers (through NBDB, logical flows, openflow flows and datapath flows) 
+that translate kubernetes feature into very specific rules that apply 
+to each packet in the network. Therefore, even though there are ways to see what OVS/OVN is doing with a particular packet, 
+there is no way to know why.
+
+We aim to solve this problem by providing a way for ovn-kubernetes to generate packet samples enriched with metadata 
+that can be easily correlated back to kubernetes objects or other human-readable pieces of information that provide 
+insights of what ovn-kubernetes is doing with a packet and why.
+
+### User-Stories/Use-Cases
+
+- As a user I want to make sure that the network policies/egress firewalls/etc. are correctly enforced in my cluster.
+- As a cluster admin I want to check why some traffic is allowed or dropped.
+
+## How to enable this feature on an OVN-Kubernetes cluster?
+
+To enable this feature, use `--observability` flag with `kind.sh` script or `--enable-observability` flag with `ovnkube` binary.
```

Comment:
I think we need another note mentioning that for now users that want to try this in kind need to bring up a cluster with the `Dockerfile.fedora.dev` dockerfile, that will use latest OVS main.  There's currently no fedora OVS build that supports psample.

Reply:
This actually changed (3 days ago).  There's no an OVS 3.4 build available in fedora:rawhide and it will be installed with the changes from https://github.com/ovn-org/ovn-kubernetes/pull/4658